sad-comparisons.py

```diff
@@ -126,30 +137,24 @@ def model_comparisons(raw_data, dataset_name, data_dir, cutoff = 9):
             
             #Convert relative likelihoods to list
             relative_likelihoods_output = relative_likelihoods.tolist() 
-            
-            # Inserts a blank in the output if the Poisson lognormal returned -inf
-            if pln_blank == 1:
-                weights_output.insert(1, '')
-                likelihood_list.insert(1, '')
-                relative_likelihoods_output.insert(1, '')
-            
-            # Inserts a blank in the output if the negative binomial exceeded the max number of iterations
-            if negbin_blank == 1:
-                weights_output.insert(2, '')
-                likelihood_list.insert(2, '')
-                relative_likelihoods_output.insert(2, '')
-                                    
+                                                
             # Format results for output
             for weight in weights_output:
                 results1 = [[site, S, N] + weights_output]
             results2 = [[site, S, N] + likelihood_list]
             results3 = [[site, S, N] + relative_likelihoods_output]
-                                        
+            results.append([site, S, N] + weights_output  + likelihood_list + relative_likelihoods_output)
+
             # Save results to a csv file:
             output1.writerows(results1)
             output2.writerows(results2)
             output3.writerows(results3)
-    
+
+    results = DataFrame(results, columns=['site', 'S', 'N', 'AICc_logseries', 'AICc_pln', 'AICc_negbin',
+                                         'AICc_zipf', 'likelihood_logseries', 'likelihood_pln',
+                                         'likelihood_negbin', 'likelihood_zipf', 'relative_ll_logseries',
+                                         'relative_ll_pln', 'relative_ll_negbin', 'relative_ll_zipf'])
+    results.to_csv(os.path.join(data_dir, dataset_name +  '_likelihood_results.csv'), index=False)
```

Comment:
My thought was that this avoided breaking down stream code since two other files rely on the 3 file structure. If/when we change how those files are ingested we can make the full switch. The reason I made the addition now was to provide a single combined output format that @davharris could easily work with for generating some of the new/different figures he's mentioned.